container_pipeline/lib/command.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+import subprocess
+
+
+def run(cmd):
+    """
+    Runs a shell command.
+
+    :param cmd: Command to run
+    :type cmd: str
+
+    :return: Command output
+    :rtype: str
+    :raises: subprocess.CalledProcessError
+    """
+    return subprocess.check_output(cmd, shell=True)
```

Comment:
For example, we have following code:
```python
def get_build_logs(self, project, build_id):
        output = run(
            'oc logs --namespace {project} build/{build_id} {suffix}'.format(
                project=project, build_id=build_id,
                suffix=self.oc_cmd_suffix))
        return output
```

What if `output` contains exception info and not expected output? If we handle it here and then subsequent methods just return the exception info till the top caller, top caller can take decision based on an `if..else` check. Correct me if I'm wrong. Nitpicking on this function since we expect it to be one of the most re-used ones across the code.